### ISSUE TITLE
Apply Job Not Working

### ISSUE THREAD

**Issue by sidd2211** [pbearne/Bullhorn2WP#2]
Hello Paul,
I am using your plugin, and configured perfectly, but the issue is, “Apply Now” was not working .

Here are the URL – http://aurumrecruitment.com/bullhorn-cv/

please can you check and let me know whats the issue with it

Looking forward to you
Thanks
Sid

**Comment by pbearne** [comment]
Hi Glad it is working for you

what do you have set for "Form Page or CV upload" option in the settings page

try to select "CV upload form" this should add a form to the job page

otherwise check that you created a page and added the shortcode form to it and set that as the apply target in "Form Page or CV upload" option

Otherwise ping me

**Comment by sidd2211** [comment]
Thanks Paul,
Its working , however i have a quick question , is it possible to customized the design as per my need to this plugin ?

Looking forward to you
Thanks
Sid

**Comment by pbearne** [comment]
it should be possible. What do you need to do? 

Happy to make changes / take pull requests

I have tried to add filters and actions (sorry not well documented yet) look in shortcodes.php

**Comment by sidd2211** [comment]
i want to design it like this UI - http://connected-talent.co.uk/career-vacancies/

let me know possibilities

Thanks
Sid

**Comment by pbearne** [comment]
yes you could make it look like the portal but it would be a lot of work it
would need a whole theme to be created

Paul

*Paul Bearne*
*Freelance Senior Full-Stack WordPress Developer*
*Cell: +1 647 780 7594*
*Home Office: +1 416 792 8894*
*E-mail: **paul@bearne.com* <pbearne@gmail.com>
*@pbearne <http://twitter.com/pbearne> *or +pbearne
<https://plus.google.com/u/0/100456877588970933323>

**Comment by sidd2211** [comment]
Hello Paul,
I am here again , a quick issue i am facing - 
i want to use search and category shortcode on the same page as shown above with job listing, i tried to use it with sidebar, however its not working, so can you please let me know , how can i show categories and search with the job listing 

Looking forward to you

Thanks
Sid

**Comment by sidd2211** [comment]
Hello Paul,
I am looking for your reply for above issue, kindly please respond me for the same 

Thanks 
Sid

**Comment by sidd2211** [comment]
Hello Paul,
Hope you are getting my messages , please reply me if you can , its last thing that i need to be done

Thanks for the help
Sid

**Comment by pbearne** [comment]
Hi

I am not totally sure what you are asking for the category are part of the
post so a normal cat list will do this.
It sounds like you will need to create a template for the jobs in you theme
so you an override the default and add the category list

If you wish me to look at this with you I will need to charge (@ $150us /
hr ) Otherwise I am not how best to help

Paul




*Paul Bearne*
*Freelance Senior Full-Stack WordPress Developer*
*Cell: +1 647 780 7594*
*Home Office: +1 416 792 8894*
*E-mail: **paul@bearne.com* <pbearne@gmail.com>
*@pbearne <http://twitter.com/pbearne> *or +pbearne
<https://plus.google.com/u/0/100456877588970933323>

**Comment by sidd2211** [comment]
Hello paul,
How does these shortcode will work ? 

[bullhorn_search]
[bullhorn_categories] 

i need to use both this shortcode with the job list here - http://aurumrecruitment.com/bullhorn-cv/

I know your time is valuable, but i am stuck only with this , so need to figure out how this will work out

Thanks
Sid

**Comment by pbearne** [comment]
These shortcodes are as I inherit the code I haven't tested them.

looking at the code

[bullhorn_search] outputs a search from filtered to just jobs

[bullhorn_categories] outputs a dropdown that links to the categories

Have you added them to a post or page?

Paul


*Paul Bearne*
*Freelance Senior Full-Stack WordPress Developer*
*Cell: +1 647 780 7594*
*Home Office: +1 416 792 8894*
*E-mail: **paul@bearne.com* <pbearne@gmail.com>
*@pbearne <http://twitter.com/pbearne> *or +pbearne
<https://plus.google.com/u/0/100456877588970933323>

**Comment by sidd2211** [comment]
Yes it's already added to that page I sent you, but it's not showing anything, not even search form

**Comment by sidd2211** [comment]
Hello Paul,
How are you today? Hope you read my last message, please let me know if you found any helpful solution for my issue .

Looking forward to you 
Thanks
Sid

**Comment by pbearne** [comment]
can you create another test page and put the short codes on that page and
send me the link

Paul

*Paul Bearne*
*Freelance Senior Full-Stack WordPress Developer*
*Cell: +1 647 780 7594*
*Home Office: +1 416 792 8894*
*E-mail: **paul@bearne.com* <pbearne@gmail.com>
*@pbearne <http://twitter.com/pbearne> *or +pbearne
<https://plus.google.com/u/0/100456877588970933323>

**Comment by sidd2211** [comment]
Here is the page - http://aurumrecruitment.com/search-job/

Search is working, however filter by category , is not

Also my concern is to use these shortcode along with Job listing 

please have a look and let me know possibilities

Thanks
Sid

**Comment by pbearne** [comment]
yes I did just a test and spotted the category didn't I have just pushed a fix 

Download and install the latest version from here (github) and try again

You can put these shortcodes in a html(text) widget in order to get them on the job listing (archive) or you will need to create an archive template and use do_shortcode() function to render the shortcodes

Paul

**Comment by sidd2211** [comment]
Ok I will do that and let you know 

Thanks for the help 
Sid

**Comment by sidd2211** [comment]
Hello Paul,
Thanks for help man, it worked now.

appreciated your help

Regards
Sid

**Comment by pbearne** [comment]
Glad to be of help feel free to test the [donate button ](https://www.paypal.com/cgi-bin/webscr?cmd=_donations&business=MZTZ5S8MGF75C&lc=CA&item_name=Wordpress%20Development%20%2f%20Paul%20Bearne&item_number=Bullhorn%20Plugin&currency_code=CAD&bn=PP%2dDonationsBF%3abtn_donateCC_LG%2egif%3aNonHosted)
